vkubelet/apiserver.go

```diff
@@ -0,0 +1,43 @@
+package vkubelet
+
+import (
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+var p Provider
+
+func ApiserverStart(provider Provider) error {
+	p = provider
+	http.HandleFunc("/", ApiServerHandler)
+	certFilePath := os.Getenv("APISERVER_CERT_LOCATION")
```

Comment:
for backward compatible, can we make sure the virtual-kubelet can still work (without exec logs support) if the environment variables are not set.